lib/Runtime/Language/SimdUtils.cpp

```diff
@@ -22,23 +47,20 @@ namespace Js
         return int32Value;
     }
 
-    int32 SIMDCheckLaneIndex(ScriptContext* scriptContext, Var lane, const int32 range)
+    int32 SIMDUtils::SIMDCheckLaneIndex(ScriptContext* scriptContext, const Var lane, const int32 range)
     {
-        int32 int32Value;
         Assert(lane != NULL);
-
-        int32Value = SIMDCheckInt32Number(scriptContext, lane);
+        int32 int32Value = SIMDCheckInt32Number(scriptContext, lane);
 
         if (int32Value < 0 || int32Value >= range)
         {
             JavascriptError::ThrowRangeError(scriptContext, JSERR_SimdLaneRangeError);
         }
-
         return int32Value;
     }
 
     // Is Number with int32 value.
-    int32 SIMDCheckInt32Number(ScriptContext* scriptContext, Var value)
+    int32 SIMDUtils::SIMDCheckInt32Number(ScriptContext* scriptContext, const Var value)
```

Comment:
Yes, these can be restricted to uint32. Changed the lane index's within helpers to be of uint32 type. Thanks.